t/rose-task-run/07-app-arch/suite.rc

```diff
@@ -46,7 +41,7 @@ sqlite3 .rose-arch.db 'SELECT * FROM sources;' | sort
 """
     [[ARCHIVE_BAD]]
         script = """rose task-run || true; RC=$?"""
-        retry delays=0
+        retry delays=P0Y
```

Comment:
Think I'd prefer PT0H or something to that effect rather than 0 Years as a delay. Probably doesn't matter though.